packages/ai/core/embed/embed-many.test.ts

```diff
@@ -57,6 +57,20 @@ describe('result.embedding', () => {
   });
 });
 
+describe('result.responses', () => {
+  it('should include responses in the result', async () => {
+    const result = await embedMany({
+      model: new MockEmbeddingModelV2({
+        maxEmbeddingsPerCall: 5,
+        doEmbed: mockEmbed(testValues, dummyEmbeddings),
+      }),
+      values: testValues,
+    });
+
+    expect(result.responses).toMatchSnapshot();
+  });
+});
```

Comment:
ideally test 2 responses with different headers/body and verify those are returned in correct order

Reply:
done! 🙏